wordpress-mobile/WordPress-Android
# Jetpack focus update bottom sheet layout

This PR updates bottom sheet layout as latest designs
- Left aligns everything
- Fixes landscape issue
- Fixes badge not clickable in dark mode

| Badge | Bottom Sheet |
|---|---|
| ![Screenshot_20220805_151404](https://user-images.githubusercontent.com/990349/183006275-cf192b19-f120-4e8d-9df9-6a23eb199ef2.png)| ![Screenshot_20220805_151239](https://user-images.githubusercontent.com/990349/183006164-7c832aa2-1eb5-462a-8677-e04c10871f59.png)|


Fixes: #17006 and #17008 

To test:

**Setup**
- Build and run
- Launch WordPress app
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find JetpackPoweredBottomFeatureConfig under Features in development Re-launch app (scroll down if necessary)

**Test**
_Find and tap on **Jetpack powered badge** on any of the following screens_

1. Home 
2. Activity Log detail
3. Reader detail
4. App Settings
5. Notifications Settings
6. Me 

- Ensure bottom sheet pops up
- Test in **light** and **dark** mode
- Test in **portrait** and **landscape** mode
- Test with Largest Font size
- Test in **RTL** languages like Arabic or Hebrew
- Test on different devices

NOTE: 
- Logo and animation will be replaced later when ready to cater for left aligned and RTL.
- Copy may not reflect the update due to translation required.  You can update values-en-rGB/strings.xml temporarily to the latest copy for testing.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing in different scenarios

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
